### PR TITLE
Add more metrics to aggregator and operator

### PR DESCRIPTION
**Pull request by Hyodar**

### CODE REVIEW

File: aggregator/aggregator.go [hunk 1]
```diff
@@ -307,10 +318,18 @@ func (agg *Aggregator) Close() error {
 func (agg *Aggregator) sendAggregatedResponseToContract(blsAggServiceResp blsagg.BlsAggregationServiceResponse) {
 	// TODO: check if blsAggServiceResp contains an err
```
Comment (taco-paco) [inline aggregator/aggregator.go:319]: Todo can be removed

File: aggregator/aggregator.go [hunk 2]
```diff
@@ -307,10 +318,18 @@ func (agg *Aggregator) Close() error {
...
 	if blsAggServiceResp.Err != nil {
+		if errors.Is(blsAggServiceResp.Err, blsagg.TaskExpiredError) {
+			agg.aggregatorListener.IncExpiredTasks()
+		}
+
```
Comment (taco-paco) [inline aggregator/aggregator.go:324]: Extra empty lines. Could be moved above

File: operator/metrics.go [hunk 3]
```diff
@@ -38,6 +73,60 @@ func (l *SelectiveRpcClientListener) OnMessagesReceived() {
 	}
 }
 
+func (l *SelectiveRpcClientListener) ObserveResendQueueSize(size int) {
+	if l.ObserveResendQueueSizeCb != nil {
+		l.ObserveResendQueueSizeCb(size)
+	}
+}
+
+func (ol *SelectiveRpcClientListener) ObserveLastCheckpointIdResponded(checkpointId uint32) {
```
Comment (taco-paco) [inline operator/metrics.go:82]:
```suggestion
func (l *SelectiveRpcClientListener) ObserveLastCheckpointIdResponded(checkpointId uint32) {
```

Receiver names a different on `SelectiveRpcClientListener`. All of the below

**Review by taco-paco — APPROVED** [review]

File: aggregator/aggregator.go (hunk 1 above)
Comment (Hyodar) [reply aggregator/aggregator.go]: Done in [b199d0f](https://github.com/NethermindEth/near-sffl/pull/159/commits/b199d0fc0ef92ecefdfa639b23d8a228401faeac)

File: operator/metrics.go (hunk 3 above)
Comment (Hyodar) [reply operator/metrics.go]: Done in [f8cf481](https://github.com/NethermindEth/near-sffl/pull/159/commits/f8cf4813c8b8932e47c02af0d1848362ca262130)

**Pull request merged**